mvp-spec/dsp-implementation.md

```diff
@@ -29,33 +29,42 @@ flowchart LR
-and the Response. The **Transmission Result** gathers all the data necessary for
-auditing a **Transmission**. The **Transmission Protocol** is designed in a way that 
-all parties that participate in the Addressable Content have all the necessary
-**Transmission Results** for auditing the **Transaction**. 
+The **Transmission Protocol** is design so that all parties that participate
+to one or many Addressable Content have all the necessary
+data for auditing a given Addressable Content.
+
+For this, **Transmission Requests** include their **"parents"** which are 
+all the **Transaction Requests** that were necessary to create the current 
+Transmission. This structure is an array to represent a single path.
+
+In a same way, **Transmission Responses** contain **"children"** which are all the 
+**Transmission Responses** that was generated thanks to the current Transmission
+*and participate or can participate to the Addressable Content*. It means that if
+an intermediary filters some potential participant in a case of an auction, the
+respective **children** are also filtred out from the Transmission. This structure
+is a tree to represent the multiplicity of the suppliers.
 
 ### Transmission Request
 
 A **Transmission Request** is always sent aside from PAF Data. It is signed so that
-we can trust that it comes from the actual **Sender**. It is associated with 
-a **Transaction** via a **Seed**.
+we can trust that it comes from the actual **Sender**. It is associated with
+**Transactions** via a **Seed**.
 
 A **Seed** is data generated by the publisher or one of its direct third parties. 
-It gathers the identity of the publisher and a **Transaction Id**. The Seed is signed
-by the actor that generated it so that it can be verified.
+It gathers the identity of the publisher, many **Transaction Id** - one per **Addressable Content**. 
+The Seed is signed by the actor that generated it so that it can be verified.
 
-A **Transmission Request** contains **Transmission Results** so that
+A **Transmission Request** contains *previous relative Transmission Requests* so that
 the **Receiver** knows all the *in progress* Transmissions that lead to the current one.
 Those **Transmission Results** are named the **parents** of the Transmission.
 
 ### Transmission Response
 
 A **Transmission Request** is mandatory only if the Receiver uses the PAF Data to participate
-to the **Addressable Content** of the **Transmission**.
+to an **Addressable Content** of the **Transmission**.
 
-A **Transmission Response** is associated to a **Transmission Response** through the 
-**Transaction Id** of its **Seed**.
+A **Transmission Response** is associated to a **Transmission Request** through a set of 
+**Transaction Id** that the Receiver is willing to participate.
 
 A **Sender** must send its own *Transmission Requests* as a Receiver and receive responses
 (or acknowledgments) before sending back its own **Transmission Response**. 
```

Comment:
(not directly related to this PR, so could be for a separate one)

Line 82 has unfinished sentence:

>The Identity Endpoint must have